kannon/master.py

```diff
@@ -25,7 +26,8 @@ def __init__(
         # kannon resources
         job_prefix: str,
         path_child_script: str = "./run_child.py",
-        env_to_inherit: Optional[List[str]] = None,
+        env_to_inherit: list[str] | None = None,
```

Comment:
[fyi]

since python 3.10, `Union[X, Y]` can be written as `X | Y`

and `from __future__ import annotations` supports it on 3.8 and 3.9